radio/src/gui/horus/bitmapbuffer.cpp

```diff
@@ -507,6 +507,20 @@ BitmapBuffer * BitmapBuffer::loadMask(const char * filename)
   return bitmap;
 }
 
+
+BitmapBuffer * BitmapBuffer::loadAndConvertMask(const char * filename, int color, int bgcolor)

```

Comment:
I am almost sure I already added this function somewhere, perhaps in default.cpp (gui/horus/themes)